Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -108,10 +108,10 @@ variable (c c' : Chain α)
 variable (f : α →o β)
 variable (g : β →o γ)
 
-instance : LE (Chain α) where le x y := ∀ i, ∃ j, x i ≤ y j
-
 lemma isChain_range : IsChain (· ≤ ·) (Set.range c) := Monotone.isChain_range (OrderHomClass.mono c)
 
+instance : LE (Chain α) where le x y := ∀ i, ∃ j, x i ≤ y j
```

Comment:
Why are you moving this instance?

Reply:
Sorry - that crept in during a merge conflict.